Plugins/Logs/Collect-WindowsEvents.ps1

```diff
@@ -235,9 +182,7 @@ $Sched_Tasks_Events = @(
         106, #Scheluded task created
         140, #Scheduled task updated
         141, #Scheduled task deleted
```

Comment:
@5ynax plugin errors on this line due to the extra comma, otherwise looks good

Reply:
should be taken care of with the latest commit. nice find

Reply:
@valrkey forgot to tag